monitoring/api/AlertSample/Program.cs

```diff
@@ -114,6 +136,21 @@ static void ListAlertPolicies(string projectId)
         }
         // [END monitoring_alert_list_policies]
 
+        static void ListNotificationChannels(string projectId)
+        {
+            var client = NotificationChannelServiceClient.Create();
+            var response = client.ListNotificationChannels(new ProjectName(projectId));
+            foreach (NotificationChannel channel in response)
+            {
+                Console.WriteLine(channel.Name);
+                if (channel.DisplayName != null)

```

Comment:
This is interesting. So the channel can have empty display name? If it's valid statement, then perhaps we may want to print something else that is always present, such as unique identifier of the channel. Otherwise for channels that don't have display name but are valid channel, you will not "list" them.

Reply:
The code already prints the channel.Name, which is always present.  If the display name is not present, the channel.Name should still tell the user that the channel is there.